{"name": "test_add2[【类型】输入特殊字符，给出提示信息2]", "status": "passed", "steps": [{"name": "步骤一：21.45和@开始计算，预期应等于参数大小超出范围，打印<ExceptionInfo for raises contextmanager>", "status": "broken", "statusDetails": {"message": "TypeError: '>' not supported between instances of 'str' and 'int'\n", "trace": "  File \"D:\\zj\\PycharmProjects\\study202208\\study0818\\test\\test_calculate.py\", line 52, in test_add2\n    result=self.cal.add(x,y)\n  File \"D:\\zj\\PycharmProjects\\study202208\\script\\calculator.py\", line 4, in add\n    if a > 99 or a < -99 or b > 99 or b < -99:\n"}, "start": 1660805207757, "stop": 1660805207757}], "attachments": [{"name": "log", "source": "ed728d14-84ea-4e97-8f7c-e9bcba094908-attachment.txt", "type": "text/plain"}, {"name": "stdout", "source": "7899c585-f7f8-4d8d-a180-687e4218b4d2-attachment.txt", "type": "text/plain"}, {"name": "stderr", "source": "2258702c-c482-4739-a5b6-2611e22426c0-attachment.txt", "type": "text/plain"}], "parameters": [{"name": "x", "value": "21.45"}, {"name": "y", "value": "'@'"}, {"name": "expect", "value": "'参数大小超出范围'"}], "start": 1660805207757, "stop": 1660805207757, "uuid": "ec8d17ea-98a9-4824-8738-6a499a1975c1", "historyId": "ad09060ab4e52a6c2d71173ec5c04aa6", "testCaseId": "c1fd4152c4894ab47d6f7448dc1c5bcf", "fullName": "study0818.test.test_calculate.TestAdd#test_add2", "labels": [{"name": "feature", "value": "加法计算模块"}, {"name": "story", "value": "加法报错用例"}, {"name": "tag", "value": "P2"}, {"name": "parentSuite", "value": "study0818.test"}, {"name": "suite", "value": "test_calculate"}, {"name": "subSuite", "value": "TestAdd"}, {"name": "host", "value": "LAPTOP-BEO6NH7O"}, {"name": "thread", "value": "7552-MainThread"}, {"name": "framework", "value": "pytest"}, {"name": "language", "value": "cpython3"}, {"name": "package", "value": "study0818.test.test_calculate"}]}